gulpfile.js

```diff
@@ -1,11 +1,38 @@
+'use strict';
+
+/*
+ * gulpfile.js
+ * ===========
+ * Rather than manage one giant configuration file responsible
+ * for creating multiple tasks, each task has been broken out into
+ * its own file in the 'gulp' folder. Any files in that directory get
+ *  automatically required below.
+ *
+ * To add a new task, simply add a new task file in that directory.
+ */
+
 var gulp = require('gulp');
-var conn = require('gulp-connect');
+var requireDir = require('require-dir');
+
+global.paths = {
+  // CSS sources
+  'css': './css/*',
```

Comment:
I think these comments are redundant, but that's just my preference.

https://sourcemaking.com/refactoring/comments